follower/consensus_follower.go

```diff
@@ -146,7 +147,15 @@ func NewConsensusFollower(
 	bindAddr string,
 	bootstapIdentities []BootstrapNodeInfo,
 	opts ...Option,
-) (*ConsensusFollowerImpl, error) {
+) (consensusFollower *ConsensusFollowerImpl, err error) {
+	// catch any unrecoverable errors encountered during initialization
+	defer func() {
+		if e := recover(); e != nil {
+			consensusFollower = nil
+			err = fmt.Errorf("%v", e)
+		}
+	}()
```

Comment:
I would put this inside `buildAccessNode`, so that it is clear which calls may actually result in panic.

You can modify `buildAccessNode` to return an error.